testdata/ansible/memcached-operator/PROJECT

```diff
@@ -1,11 +1,15 @@
 domain: example.com
 layout: ansible.sdk.operatorframework.io/v1
+plugins:
```

Comment:
Hmm this field was deliberately marshaled last so it would always be at the bottom. There might be a regression upstream @Adirio.

Reply:
It was marshalled separately to do some config/v2 logic and with the `config.Config` interface with different implementations for each version that logic is no longer needed so it is now being marshalled at the same time that the rest. I can't find any reason why we should keep this last, but if you have any we can handle this in two steps again.

Reply:
Mainly because that key contains multiple other configs that could grow very large down the line. Its a cosmetic thing. I guess I'm fine leaving it as-is, but that was the original intent of marshaling it last.

Reply:
Okey I see your point. Luckyly, loading doesn't care about the order and each time you call a plugin the file is re-written, so if we decide to go back to keeping plugins as the last key, it will be backwards compatible with every project. That means that we can tackle this issue when we find it.